src/plot.jl

```diff
@@ -202,7 +204,11 @@ function _plot!(plt::Plot, plotattributes::KW, args::Tuple)
     # --------------------------------
     # Plot/Subplot/Layout setup
     # --------------------------------
+
+    # 2.5 seconds
     _plot_setup(plt, plotattributes, kw_list)
+
+    # 6 seconds
```

Comment:
You get to 6 seconds by the time it's here due to inference issues in the argument handling chain. Profile it to see.